ui/raidboss/data/05-shb/trial/diamond_weapon.js

```diff
@@ -0,0 +1,147 @@
+      durationSeconds: 8,
+      outputStrings: sharedOutputStrings,
+      alertText: (data, _, output) => output.teleportWest(),
+    },
+    {
+      id: 'Diamond Claw Swipe West',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5F9B', capture: false }),
+      delaySeconds: 2,
+      durationSeconds: 8,
+      outputStrings: sharedOutputStrings,
+      alertText: (data, _, output) => output.teleportEast(),
+    },
+    {
+      id: 'Diamond Photon Burst',
+      netRegex: NetRegexes.headMarker({ id: '0057' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 4,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Flare on YOU',
+          de: 'Flare auf DIR',
+          fr: 'Brasier sur VOUS',
+          ja: '自分にフレア',
+          cn: '核爆点名',
+          ko: '플레어 대상자',
+        },
+      },
+    },
+    {
+      id: 'Diamond Diamond Flash',
+      netRegex: NetRegexes.ability({ source: 'The Diamond Weapon', id: '5779' }),
+      durationSeconds: 5,
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Diamond Auri Cyclone',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FE6', capture: false }),
+      delaySeconds: 0.5,
+      durationSeconds: 6,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'Diamond Airship\'s Bane',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FE8', capture: false }),
+      durationSeconds: 5,
+      outputStrings: sharedOutputStrings,
+      alertText: (data, _, output) => output.teleportEast(),
+    },
+    {
+      id: 'Diamond Outrage',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FD7', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 4,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Diamond Auri Doomstead',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FD8' }),
+      durationSeconds: 4,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Diamond Vertical Cleave',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FE5', capture: false }),
+      durationSeconds: 5,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'Diamond Diamond Shrapnel',
+      netRegex: NetRegexes.headMarker({ id: '00C5' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 7,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Diamond Shrapnel on YOU',
+        },
+      },
+    },
+    {
+      id: 'Diamond Articulated Bits',
+      netRegex: NetRegexes.ability({ source: 'The Diamond Weapon', id: '5FA9', capture: false }),
+      durationSeconds: 4,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Avoid hands',
```

Comment:
This is a microbikeshed, but maybe `Avoid Bits` (since folks know what bits are, and this is summoning bits?) or even `Avoid lasers` (if you want to say what they're doing rather than what they are)?